pkgs/development/tools/jdt-language-server/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, jdk
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jdt-language-server";
+  version = "0.67.0";
+  timestamp = "202012170459";
+
+  src = fetchurl {
+    url = "https://download.eclipse.org/jdtls/milestones/${version}/jdt-language-server-${version}-${timestamp}.tar.gz";
+    sha256 = "11qzwf2l737hdv1g7bfcrmdxcrl7g9z2k9nhy9lknlabmhq4mcdl";
+  };
+
+  sourceRoot = ".";
```

Comment:
I rarely see this specified; I don't think it's necessary?

Reply:
Gave it a shot, but the build errors out with "unpacker produced multiple directories"

Reply:
It is necessary because the unpack phase produces multiple directories

Reply:
Thanks for clarifying that @jlesquembre